rpm/el9/repodata/repomd.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<repomd xmlns="http://linux.duke.edu/metadata/repo" xmlns:rpm="http://linux.duke.edu/metadata/rpm">
+ <revision>1677169356</revision>
+</repomd>
```

Comment:
You can delete this, it will be generated.

Reply:
Done.